src/GitVersionTask/GenerateGitVersionInformation.cs

```diff
@@ -1,42 +0,0 @@
-namespace GitVersionTask
-{
-    using System.IO;
-    using GitVersion;
-    using GitVersion.Helpers;
-    using Microsoft.Build.Framework;
-
-    public class GenerateGitVersionInformation : GitVersionTaskBase
```

Comment:
It would be nice if Git detected this as a move and not delete/add.

Reply:
Actually that's what you see in GitHub the changes made in the PR. If you look closer in each of the commits there are separate commits, and from one commit to another you can see in the history the rename

Reply:
you can see the commit 046621700e94f6fb901c9fdd866995558c28fa32 that does the rename

Reply:
Ah, ok. Not the easiest thing in the world to do code reviews on a mobile phone. :smile:

Reply:
Agree